sys/log/full/src/log_fcb.c

```diff
@@ -772,9 +816,7 @@ log_fcb_copy(struct log *log, struct fcb *src_fcb, struct fcb *dst_fcb,
 {
     struct fcb_entry entry;
     int rc;
-
```

Comment:
Empty line between local variable declarations and function body improves readability. Don't remove it.

Reply:
Agreed. Done.